src/vs/workbench/browser/parts/globalCompositeBar.ts

```diff
@@ -375,6 +379,12 @@ export class AccountsActivityActionViewItem extends AbstractGlobalActivityAction
 
 	//#region overrides
 
+	protected override updateLabel(): void {
+		super.updateLabel();
+		this.label.classList.toggle('show-avatar', !!this.accountForAvatar);
+		this.label.style.setProperty('background-image', this.accountForAvatar?.iconPath ? `url("${URI.from(this.accountForAvatar.iconPath).fsPath}")` : '');
```

Comment:
Why is this `fsPath` ? Is the file stored on disk? How does this work in web version then?